tests/test_dispatcher.py

```diff
@@ -174,6 +174,15 @@ def test_error_handler_that_raises_errors(self, dp):
 
         assert self.count == 1
 
+    def test_default_run_async(self, monkeypatch, dp):
```

Comment:
mh, `dp.bot` doesn't have the default settings, iisc, which explains why the coverage is still failing. maybe set those manually here (but make sure to delete them at the end of the test)